code/go/0chain.net/smartcontract/storagesc/validator_eventdb.go

```diff
@@ -0,0 +1,37 @@
+package storagesc
+
+import (
+	"encoding/json"
+	"fmt"
+
+	cstate "0chain.net/chaincore/chain/state"
+	"0chain.net/chaincore/transaction"
+	"0chain.net/smartcontract/dbs/event"
+)
+
+func writeMarkerToValidationNode(vn *ValidationNode) *event.Validator {
+	return &event.Validator{
+		ValidatorID: vn.ID,
+		BaseUrl:     vn.BaseURL,
+		// TO-DO
```

Comment:
What is the todo over here?

Reply:
Update the maker to update the stake, since Validation Node doesn't have the stake info.